src/wp-settings.php

```diff
@@ -268,6 +268,8 @@
 require ABSPATH . WPINC . '/html-api/class-wp-html-stack-event.php';
 require ABSPATH . WPINC . '/html-api/class-wp-html-processor-state.php';
 require ABSPATH . WPINC . '/html-api/class-wp-html-processor.php';
+require ABSPATH . WPINC . '/class-wp-block-scanner.php';
+require ABSPATH . WPINC . '/html-api/css-helpers.php';
```

Comment:
I don't think this file is included in the PR. Should it be the block processor file instead?

Reply:
this has been removed